src/interfaces/item-tag-task-manager.ts

```diff
@@ -17,4 +16,11 @@ export interface ItemTagTaskManager<A extends Actor = Actor> {
   createDeleteTask(actor: A, objectId: string, extra?: unknown): Task<A, ItemTag>;
 
   createGetOfItemTask(actor: A, item: Item): Task<A, ItemTag[]>;
+  createGetOfItemTaskSequence(member: A, itemId: string): Task<Actor, unknown>[];
```

Comment:
info: what's the difference between `createGetOfItemTaskSequence` and `createGetOfItemTask` ?